introduction.md

```diff
@@ -1 +1,2 @@
-This is my introduction - Code School JavaScript is my favorite
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?